Makefile

```diff
@@ -1,5 +1,5 @@
 all    :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1000000 dapp --use solc:0.6.12 build
 clean  :; dapp clean
 test   :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1000000 dapp --use solc:0.6.12 test -v
-deploy-mainnet :; make && dapp create IlkRegistry 0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B 0xa5679C04fc3d9d8b0AaB1F0ab83555b301cA70Ea 0x65C79fcB50Ca1594B025960e539eD7A9a6D434A3
-deploy-kovan :; make && dapp create IlkRegistry 0xbA987bDB501d131f766fEe8180Da5d81b34b69d9 0xdDb5F7A3A5558b9a6a1f3382BD75E2268d1c6958 0x3a042de6413eDB15F2784f2f97cC68C7E9750b2D
+deploy-mainnet :; make && dapp create IlkRegistry 0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B TODO:DOG 0xa5679C04fc3d9d8b0AaB1F0ab83555b301cA70Ea 0x65C79fcB50Ca1594B025960e539eD7A9a6D434A3
+deploy-kovan :; make && dapp create IlkRegistry 0xbA987bDB501d131f766fEe8180Da5d81b34b69d9 0x121D0953683F74e9a338D40d9b4659C0EBb539a0 0xdDb5F7A3A5558b9a6a1f3382BD75E2268d1c6958 0x3a042de6413eDB15F2784f2f97cC68C7E9750b2D
```

Comment:
Not sure what the use of this target is if the "correct" way to deploy to Kovan is to use the script...maybe this target should actually just execute `scripts/deploy-kovan.sh`?

Reply:
This was preexisting. I'm not sure the correct way is to use the script. It's just helpful to have in case you want to fully populate it here.